### ISSUE TITLE
Submission and Autograding?

### ISSUE THREAD

**Issue by emmannyyy** [nus-cs2030/2223-s2#215]
# Summary
For PA1 must we submit our code onto codecrunch before time is out? How is marking conducted?

## Description 
For PA1, will we be required to compile and submit our code for each level? Or will it automatically be "saved" like our usual lab practices? Also, if we don't submit, how would we know if we pass the individual levels respectively?

I understand that we are graded for our initial attempt during the lab and subsequently penalised for changes made during that 1 week. My question is: Are we meant to submit the files for grading per level? is that recommended?

**Comment by taugwa** [comment]
it should be submitted as long as you :w your code after youre done...i think they mark it as a whole and not by each level

**Comment by territxl** [comment]
yup, i think its marked based on how similar your submitted code (within the 2 hours) is with your final submission
